### PR TITLE
made update in preparation for course

### PR DESCRIPTION
**Pull request by TongruiLi**
Following changes has been applied:
- Added in continuous auto mode
- updated readme
- split make to sim and hardware
- made map symbolic link
- updated f1tenth to ut_automata

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,16 @@ After adding these lines you will need to either relog into the computer or run:
...
+   cd ut_automata
+   make -j
+   ```
+
+2. Alternatively if you have accidently forgot the ```--recurse-submodule``` flag during cloning phase, you can use the following steps instead:
+   ```
+   git clone https://github.com/ut-amrl/ut_automata.git
+   cd ut_automata
+   git submodule init
+   git submodule update
```
Comment (jarrettHoltz) [inline README.md:39]: `git submodule update --init ` <- Will do this in one line

**Review by jarrettHoltz — APPROVED** [review]
Looks fine to me.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# f1tenth_course
-Repository for F1/10 courses
+# ut_automata
+Repository for ut_automata courses
```
Comment (joydeep-b) [inline README.md]: UT AUTOmata

File: scripts/start_ros.sh [hunk 3]
```diff
@@ -4,13 +4,6 @@
 source /opt/ros/melodic/setup.bash
```
Comment (joydeep-b) [inline scripts/start_ros.sh]: Remove this file.

**Review by joydeep-b — APPROVED** [review]

**Pull request merged**